landing-strip/core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -0,0 +1,133 @@
+package com.novoda.landingstrip;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.HorizontalScrollView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+public class LandingStrip extends HorizontalScrollView implements Scrollable {
+
+    private static final int TAG_KEY_POSITION = R.id.tag_key_position;
+
+    private final Attributes attributes;
+    private final LayoutInflater layoutInflater;
+    private final Paint indicatorPaint;
+    private final State state;
+    private final LinearLayout tabsContainer;
+    private final IndicatorCoordinatesCalculator indicatorCoordinatesCalculator;
+
+    private ViewPager viewPager;
+
+    public LandingStrip(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        setFillViewport(true);
+        setWillNotDraw(false);
+        setHorizontalScrollBarEnabled(false);
+
+        this.attributes = Attributes.readAttributes(context, attrs);
+        this.layoutInflater = LayoutInflater.from(context);
+        this.indicatorPaint = new Paint();
+        this.state = new State();
+        this.tabsContainer = new LinearLayout(context);
+        this.indicatorCoordinatesCalculator = IndicatorCoordinatesCalculator.newInstance();
+
+        state.updateDelegateOnPageListener(new ViewPager.SimpleOnPageChangeListener());
+        state.updatePosition(0);
+        state.updatePositionOffset(0);
```

Comment:
:fried_shrimp: either for me - just not here. I don't follow why position and positionOffset would ever be non-zero when you create a State object.